WTraining/app/services/weet.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const { requestGet } = require('../helpers/request');
+const logger = require('../logger');
+
+const URL = process.env.ULR_WEET;
```

Comment:
Add this variable (ULR_WEET) in config/index.js file and retrieve the value from there